packages/stryker-typescript/test/unit/mutator/StringLiteralMutatorSpec.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from 'chai';
+import { expectMutation } from './mutatorAssertions';
+import StringLiteralMutator from '../../../src/mutator/StringLiteralMutator';
+
+describe('StringLiteralMutator', () => {
+
+  let sut: StringLiteralMutator;
+
+  beforeEach(() => {
+    sut = new StringLiteralMutator();
+  });
+
+  it('should have name "StringLiteral"', () => {
+    expect(sut.name).eq('StringLiteral');
+  });
+
+  it('should mutate a string literal with double quotes', () => {
+    expectMutation(sut, 'const b = "Hello world!";', 'const b = "";');
+  });
+
+  it('should mutate a string literal with single quotes', () => {
+    expectMutation(sut, 'const b = \'Hello world!\';', 'const b = "";');
+  });
+
+  it('should mutate a template string', () => {
+    expectMutation(sut, 'const b = `Hello world!`;', 'const b = "";');
+  });
+
+  it('should mutate a template string referencing another variable', () => {
+    expectMutation(sut, 'const a = 10; const b = `${a} mutations`;', 'const a = 10; const b = "";');
+  });
+
+  it('should mutate a template string referencing another variable', () => {
+    expectMutation(sut, 'const a = 10; const b = `mutations: ${a}`;', 'const a = 10; const b = "";');
+  });
+
+  it('should mutate a template string referencing another variable', () => {
+    expectMutation(sut, 'const a = 10; const b = `mutations: ${a} out of 10`;', 'const a = 10; const b = "";');
+  });
+
+  it('should mutate import statements', () => {
```

Comment:
Is there a way to not mutate import strings? There is no point, it will always result in a transpile error.

Reply:
I tried few things but as of right now `node.parent` is always undefined, for all nodes. If it was not then theoretically, `node.parent.kind === ts.SyntaxKind.ImportDeclaration` and we could drop out in that case.

Reply:
Hmm until now we didn't need the `parent` property, but i can see why we need it here. I've created a separate PR for it here: https://github.com/stryker-mutator/stryker/pull/463. Once that's merged you should be able to use `node.parent`.